spec/features/workflow/schedule_set_datetime_passes_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+RSpec.feature "Set scheduled publishing date and time passes before scheduling" do
+  scenario do
+    given_there_is_an_edition_with_a_scheduled_publishing_date_and_time_set_before_now
+    when_i_visit_the_summary_page
+    then_i_cannot_see_a_schedule_link
+  end
+
+  def given_there_is_an_edition_with_a_scheduled_publishing_date_and_time_set_before_now
```

Comment:
This method name feels pretty big - could it be "given_there_is_an_edition_with_a_past_scheduled_publishing_datetime" (which admittedly isn't much shorter) 

I thought maybe you could get away with `given_there_is_an_edition_scheduled_for_the_past` but that felt a bit wrong as status isn't scheduled.

Reply:
Could we split the step in two?

Reply:
Not sure it makes sense to split it in two when it's setting up the edition to be in the state that we need it to be. Could do `given_there_is_an_edition_with_past_scheduled_publishing_datetime`?

Reply:
How about `given_there_is_an_edition_to_be_scheduled_in_the_past`?

Reply:
I'm not sure that makes it clear what exactly is in the past.

Reply:
I think it's a bit of a compromise with readability. For example, we've got steps like `when_i_go_to_the_tags_page`, which leave it ambiguous as to how that happens. I think it's harder to understand what 'past scheduled publishing datetime' means - one too many adjectives for me ;-).